tools/test/examples/examples_lib.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
Are we using already py3 in the new files (most of the files in the codebase have py2 still here)?

Reply:
I was just checking, to be aware of this and won't ask anymore.

Reply:
I think it is depends on @OPpuolitaival and CI configurations. If CI ok to take it then we should be fine, because this script only used by CI at moment

Reply:
Python 2 seems to be the version supported.